# request params for grasp planners (not all of these must be filled in)

# color and depth images
sensor_msgs/Image color_image
sensor_msgs/Image depth_image
sensor_msgs/Image seg_image

# camera info and intrinsics
sensor_msgs/CameraInfo camera_info

# pointcloud and viewpoint
sensor_msgs/PointCloud2 cloud # is in world frame
geometry_msgs/PoseStamped view_point
geometry_msgs/Pose aruco_board
bool grasp_filter_flag

# number of candidates to return
int16 n_of_candidates

---

# response params
BenchmarkGrasp[] grasp_candidates